spec/spec_helper.rb

```diff
@@ -2,6 +2,8 @@
 Dotenv.load '.env.test.local', '.env.test', '.env'
 
 require 'rspec/json_expectations'
+require "dfe/analytics/big_query_api"
+require "dfe/analytics/big_query_legacy_api"
```

Comment:
@ericaporter - Maybe we should be adding these lines to `dfe/analytics/testing.rb`